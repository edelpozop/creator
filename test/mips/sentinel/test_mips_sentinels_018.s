
#
# ARCOS.INF.UC3M.ES
# BY-NC-SA (https://creativecommons.org/licenses/by-nc-sa/4.0/deed.es)
#


.text

	main: 
			
			li $sp 5
            jal test

            # exit
            li $v0 10
            syscall


	test:
            # crear "stack frame" para $ra, $fp y una variable local
            
			addi $sp 4

            # return $v0
           jr $ra
