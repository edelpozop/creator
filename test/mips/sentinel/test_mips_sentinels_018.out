

Error at line 29 (addi $reg2 $reg1 val):
Incorrect instruction syntax for 'addi $reg2 $reg1 val'

  28 undefined
->29 undefined

Not executed
keyboard[0x0]:''; display[0x0]:''; 

